lib/matplotlib/backends/backend_gtk3.py

```diff
@@ -36,17 +36,26 @@
 backend_version = "%s.%s.%s" % (
     Gtk.get_major_version(), Gtk.get_minor_version(), Gtk.get_micro_version())
 
-try:
-    _display = Gdk.Display.get_default()
-    cursord = {  # deprecated in Matplotlib 3.5.
-        Cursors.MOVE:          Gdk.Cursor.new_from_name(_display, "move"),
-        Cursors.HAND:          Gdk.Cursor.new_from_name(_display, "pointer"),
-        Cursors.POINTER:       Gdk.Cursor.new_from_name(_display, "default"),
-        Cursors.SELECT_REGION: Gdk.Cursor.new_from_name(_display, "crosshair"),
-        Cursors.WAIT:          Gdk.Cursor.new_from_name(_display, "wait"),
-    }
-except TypeError as exc:
-    cursord = {}  # deprecated in Matplotlib 3.5.
+
+@functools.lru_cache(None)
```

Comment:
```suggestion
# module-level depredations
@functools.lru_cache(None)
```

Since we don't make a dedicated function which could give the code segment a name, let's at least use a comment as a standard convention for explaining what this is for. - Also for the other uses of this pattern.

Reply:
How would you want that to be for `matplotlib/__init__.py`, which also uses `__getattr__` for `__version__`?

Reply:
For simplicity, you can leave that without a comment. Part of the intent of the comment is to easily see that the whole construct serves the deprecation and can be removed when expiring. That's not the case for the `matplotlib.__init__.py` `__getattr__`.

Reply:
Actually, I just put the comment in the `if` block.